assets/scripts/index.js

```diff
@@ -2,15 +2,23 @@ const catApi="https://api.thecatapi.com/v1/breeds";
 const allcats=document.getElementById("allcats");
 const submitButton=document.querySelector(".submitButton");
 const form=document.getElementById("form");
+const allCatsButton=document.querySelector(".allCatsButton");
+let catBreeds; 
 
 
 
-submitButton.addEventListener("click", (e) => {
+submitButton.addEventListener("click", async (e) => {
     e.preventDefault();
     getUserInputs();
     console.log(userValues);
-})
 
+    let result = await getAllCatBreeds();
+    if(result){
+        console.log(result.data);
+        catBreeds = result.data;
+        //Code for displaying the cats based on the input search parameters goes here.
+    }
+});
 
 function getUserInputs() {
     const inputs=form.elements;
```

Comment:
Recommend keeping spaces between assignments and equality comparisons not only because it's consistent with line 18 but for readability's sake too.

`const inputs = form.elements;`

The comments so far would be warned about with tools like ESLint (it can also format files for you).